config/configuration.yml.example

```diff
@@ -94,6 +94,12 @@
 #
 # === More configuration options
 #
+# Disable default module:
+#
+# By default user may choose which modules can be disabled
+# this can be done in lib/open_project/configuration.rb
+# disabled_modules:[list_of_modules_name]
```

Comment:
This is not the structure the user has to specify the setting in. Because it is an Array, it would be:

```
disabled_modules:
  - module_name_1
  - module_name_2
```